packages/core/src/generatePolicy.js

```diff
@@ -160,6 +213,7 @@ function createModuleInspector(opts = {}) {
     if (includeDebugInfo) {
       const moduleDebug = debugInfo[moduleRecord.specifier]
       moduleDebug.sesCompat = {
+        // XXX: I don't think this is needed, since it appears we overwrite all properties
```

Comment:
this is just for the debug file. what do you mean about overwriting all props?

Reply:
`compatWarnings` has props `primordialMutations`, `strictModeViolations` and `dynamicRequires`, all of which are overwritten below.  I don't think there are other props.

Reply:
(meaning that `...compatWarnings` is unnecessary. at least, I think)